pom.xml

```diff
@@ -1428,6 +1426,10 @@
             <groupId>org.codehaus.groovy</groupId>
             <artifactId>groovy-all</artifactId>
           </exclusion>
+          <exclusion>
```

Comment:
I see, hive-exec pulls this in? then it's fine to exclude, sure. Sounds like it should have been there before if so, so just checking that this is required now.